### ISSUE TITLE
Can't find worker.js file.

### ISSUE THREAD

**Issue by essamamdani** [ZazerConer/info-tube#1]
Are you planing to opensource it?

**Comment by ZazerConer** [comment]
Yes! Now I have added it [here](https://github.com/ZazerConer/info-tube/blob/main/worker.js)

**Comment by essamamdani** [comment]
Thanks brother. I appreciate

**Comment by essamamdani** [comment]
I just want to know why sometime this url is not being proceed and having error (failed with status: 429).

https://info-tube.zaco.workers.dev/watch?v=PYKVrW3L1cc

**Comment by ZazerConer** [comment]
You're right, but sometimes it's ok and sometimes it's an error. I'll check later, for now I'm busy. Thanks for reminding me.

**Comment by ZazerConer** [comment]
Updated!

Using cache is the best option so far.

Cached to request: 5 seconds [#29](https://github.com/ZazerConer/info-tube/blob/077a9a7b67f839832c5d4129cb898d58ee5ba4d8/worker.js#L29)
Cached in the browser: 1500 seconds (25 minutes) [#62](https://github.com/ZazerConer/info-tube/blob/077a9a7b67f839832c5d4129cb898d58ee5ba4d8/worker.js#L62)

As long as the request response status is 200 (ok), it can be requested multiple times after being cached.

Each request is cached for 5 seconds but will be duplicated for the next request if the url is not specified.

For the cache in the browser will happen for 25 minutes and keep repeating to be used many times but after that it will be cached again if there is a new request.

If the first request works but the second time gets an error, wait a few seconds and make the request again.
